schema/types/isolator.go

```diff
@@ -35,13 +35,23 @@ func (is *Isolators) GetByName(name string) *Isolator {
 	return nil
 }
 
+// returns a set of isolators that were not recognized
```

Comment:
Unrecognized returns a set of isolators that are not defined in the specification.